style.css

```diff
@@ -1,43 +1,79 @@
 * {
-	text-align: center;
+  text-align: center;
+}
+
+body {
+  display: grid;
+  grid-template-columns: auto;
 }
 
 .heading {
-	font-family: Arial;
-	margin-top:40vh;
+  font-family: Arial;
+  margin-top: 40vh;
 }
 
-.URL-input, .convert-button {
-	font-size:1.3em;
-	padding:5px 10px;
+.URL-input,
+.convert-button {
+  font-size: 1.3em;
+  padding: 5px 10px;
 }
 
 .URL-input {
-	border-radius:4px 0px 0px 4px;
-	width:30em;
-	text-align: left;
-	border:2px solid #EEEEEE;
-	background: #EEEEEE;
-	outline:none;
+  margin: auto;
+  border-radius: 4px 0px 0px 4px;
+  width: 30em;
+  text-align: left;
+  border: 2px solid #eeeeee;
+  background: #eeeeee;
+  outline: none;
 }
 
 .URL-input:focus {
-	border:2px solid #0485ff;
+  border: 2px solid #0485ff;
 }
 
 .convert-button {
-	border:2px solid #0485ff;
-	background: #0485ff;
-	color:white;
-	transition: 0.15s;
+  margin: 2% auto;
+  border: 2px solid #0485ff;
+  background: #0485ff;
+  color: white;
+  transition: 0.15s;
 }
 
 .convert-button:hover {
-	background: #016acc;
-	border-color: #016acc;
+  background: #016acc;
+  border-color: #016acc;
 }
 
 #mp3 {
-	border-radius:0px 4px 4px 0px;
+  margin: 2% auto;
+  border-radius: 0px 4px 4px 0px;
+}
+
+@media only screen and (max-width: 600px) {
+  body {
+    display: grid;
+    grid-template-columns: auto;
+    justify-content: center;
+  }
+  .URL-input {
+    margin: auto;
+    width: 100%;
+  }
+  .convert-button {
+    margin: 7% auto;
+    width: 100%;
+  }
+}
+
+body {
+  color: snow;
+  background: #000;
 }
```

Comment:
I don't really like the fully black background with the design that is implemented already. I do prefer dark themes generally but if we are going to change the background then we will have to change the design of the elements (You can open another pull request for a design change)